src/testing/fixtures/algorand-fixture.ts

```diff
@@ -92,7 +92,7 @@ export function algorandFixture(fixtureConfig?: AlgorandFixtureConfig, config?:
     const transactionLoggerAlgod = transactionLogger.capture(algod)
     algorand = AlgorandClient.fromClients({ algod: transactionLoggerAlgod, indexer, kmd })
     const acc = await getTestAccount({ initialFunds: fixtureConfig?.testAccountFunding ?? algos(10), suppressLog: true }, algorand)
-    algorand.setSignerFromAccount(acc).setSuggestedParamsTimeout(0)
+    algorand.setSignerFromAccount(acc).setSuggestedParamsCacheTimeout(0)
```

Comment:
@SilentRhetoric while reviewing the method names we realised this method name was more descriptive as `CacheTimeout`